operator-framework/src/test/java/io/javaoperatorsdk/operator/KubernetesResourceStatusUpdateIT.java

```diff
@@ -30,15 +30,14 @@
   @Test
   public void testReconciliationOfNonCustomResourceAndStatusUpdate() {
     var deployment = operator.create(Deployment.class, testDeployment());
-    await().atMost(10, TimeUnit.SECONDS).untilAsserted(() -> {
+    await().atMost(15, TimeUnit.SECONDS).untilAsserted(() -> {
```

Comment:
Why is it taking longer now, though?

Reply:
just tring somehing, just on one kubernetes version in tests sometimes fails

Reply:
It seems that the problem is with namespace deletion, not with the actual test logic.